indexer/postgres/README.md

```diff
@@ -0,0 +1,41 @@
+| `Uint16Kind`        | `INTEGER`                  |                                                                                                                                                                                 |
+| `Uint32Kind`        | `BIGINT`                   |                                                                                                                                                                                 |
+| `Uint64Kind`        | `NUMERIC`                  |                                                                                                                                                                                 |
+| `Float32Kind`       | `REAL`                     |                                                                                                                                                                                 |
+| `Float64Kind`       | `DOUBLE PRECISION`         |                                                                                                                                                                                 |
+| `IntegerStringKind` | `NUMERIC`                  |                                                                                                                                                                                 |
+| `DecimalStringKind` | `NUMERIC`                  |                                                                                                                                                                                 |
+| `JSONKind`          | `JSONB`                    |                                                                                                                                                                                 |
+| `Bech32AddressKind` | `TEXT`                     | addresses are converted to strings with the specified address prefix                                                                                                            |
+| `TimeKind`          | `BIGINT` and `TIMESTAMPTZ` | time types are stored as two columns, one with the `_nanos` suffix with full nanoseconds precision, and another as a `TIMESTAMPTZ` generated column with microsecond precision |
+| `DurationKind`      | `BIGINT`                   | durations are stored as a single column in nanoseconds                                                                                                                          |
+| `EnumKind` | `<module_name>_<enum_name>` | a custom enum type is created for each module prefixed with the module name it pertains to                                                                                     |
+
+
```

Comment:
It would be great to add somewhere how to actually use it.

Reply:
Was going to add something once the indexer manager is all wired up and we can see how app.toml works (see #20647)